codecarbon/core/emissions.py

```diff
@@ -24,7 +24,9 @@ def __init__(
         self._data_source = data_source
         self._co2_signal_api_token = co2_signal_api_token
 
-    def get_cloud_emissions(self, energy: Energy, cloud: CloudMetadata) -> float:
+    def get_cloud_emissions(
+        self, energy: Energy, cloud: CloudMetadata, geo=None
+    ) -> float:
         """
         Computes emissions for cloud infra
         :param energy: Mean power consumption of the process (kWh)
```

Comment:
add the doc of the param geo here. It is a boolean I guess?

Reply:
No, it's a GeoMetadata, I've updated the typing and the docstring.
Thanks for noticing it.